.travis.yml

```diff
@@ -11,6 +11,8 @@ install:
   - sudo apt-get -qq update
   - sudo apt-get -qq install monodevelop-current
   - sudo apt-get -qq install g++-4.8
+  - sudo apt-get -qq install node
+  - sudo npm install -g typescript
```

Comment:
@vheon @Valloric the version of `node` installed doesn't come with `npm`. Ideas?